lib/src/extensions.dart

```diff
@@ -432,6 +432,19 @@ extension FlowX<T> on Flow<T> {
       await strategy.handle(cacheFlow, this, collector);
     });
   }
+  
+  /// Filters out null from emitted values
+  ///
+  /// Example:
+  /// ```dart
+  ///   flow([1, 2, 3, null, 4, null]).filterNotNull()
+  ///     .collect(print); // This will print only even numbers (1, 2, 3, 4)
+  /// ```
+  Flow<T> filterNotNull() => flow((collector) async {
+    await collect((value) async {
+      if (value != null) collector.emit(value);
+    });
+  });
```

Comment:
why don't we just use the existing filter method?

```dart
Flow<T> filterNotNull() => this.filter((value) => value != null);
```

Reply:
Oh alright that will work